docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -99,6 +99,8 @@ The following table shows a configuration option's name, type, and the default v
 |[variables-hash-bucket-size](#variables-hash-bucket-size)|int|128|
 |[variables-hash-max-size](#variables-hash-max-size)|int|2048|
 |[upstream-keepalive-connections](#upstream-keepalive-connections)|int|32|
+|[upstream-keep-alive](#upstream-keep-alive)|int|60s|
```

Comment:
Could you stick to the existing naming convention of `keepalive` rather than `keep-alive`? That'll also be consistent with Nginx's naming of respective directives.

Reply:
> |int|60s|

I think the default here should be `60` instead of `60s` no? For example `keep-alive` setting accepts only integer, and when you provide something like `60s` it logs an error and uses default value.

Reply:
Nice catch! Forgot to change this part.